docs/getting-started/installation.md

```diff
@@ -57,6 +57,34 @@ docker run -d ...
 Use a 3rd party updating mechanism such as [Watchtower](https://github.com/containrrr/watchtower) or [Ouroboros](https://github.com/pyouroboros/ouroboros) to keep Overseerr up-to-date automatically.
 {% endhint %}
 
+## Native Linux
+{% hint style="info" %}
+These instructions are provided by the community and are not necessarily currently supported. Please make sure you know what you're doing if you're willing to go down this path.
+{% endhint %}
+
+Please note, these instructions are targetted at `apt`-based systems, but modify these instructions to fit your own
+```bash
+sudo su                                   # You REALLY need to know what you're doing
+# Install nodejs your OWN way https://nodejs.org/en/download/package-manager/
+curl -sL https://deb.nodesource.com/setup_12.x | -E bash -
+apt install -y nodejs
+apt install sqlite3 libsqlite3-dev        # This might be necessary if you're failing to install due to sqlite3
+cd /opt
+git clone https://github.com/sct/overseerr.git
+cd overseerr
+nano next.config.js                       # Only if you need to change some pre-build values 
```

Comment:
No one should really every touch this file. Any possible configuration will be done through environment variables. (Otherwise might have merge conflicts in the future)

Reply:
```suggestion
```